tests/prepare/feature/data/check-repos.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -ex
+
+pkgmgr=$(dnf --version > /dev/null 2>&1 && echo dnf || echo yum)
+
+STATE=${STATE:-"enabled"}
+REPOSITORIES=${REPOSITORIES:-"epel epel-debuginfo epel-source"}
+
+for reponame in $REPOSITORIES; do
+    if [[ "$STATE" == "enabled" ]]; then
+        $pkgmgr repoinfo "$reponame" | grep -E "^Repo-status[[:space:]]+: enabled"
+    else
+        ($pkgmgr repoinfo "$reponame" | grep -E "^Repo-status[[:space:]]+: disabled") || \
+            ($pkgmgr repoinfo "$reponame" | grep -E '^Total packages: 0')
```

Comment:
What is the point of this check? Seems that `dnf repoinfo` show this even for non-existent repos:

```
# dnf repoinfo nonsense
Last metadata expiration check: 0:03:09 ago on Thu Feb  1 13:22:23 2024.
Total packages: 0
```

This can wait for @happz once he's back.

Reply:
Ok, I've read the playbook properly. I see that repo is not installed at all.